framework-plugins/lisk-framework-http-api-plugin/swagger.yml

```
@@ -66,3 +47,768 @@ info:
+        description: |
+          The total votes received by the delegate.
+          Represents the total amount of token that the delegates voters voted this delegate.
+      username:
+        type: string
+        example: genesis_01
+        description: |
+          The delegates username.
+          A delegate chooses the username by registering a delegate on the network.
+          It is unique and cannot be changed later.
+      nextForgingTime:
+        type: number
+        example: 4368793
+        description: |
+          Returns the unix timestamp in which the forger will be able to forge the next block.
+          Each slot has a timespan of 10 seconds.
+
+  Peer:
+    type: object
+    required:
+      - ipAddress
+      - port
+      - networkID
+      - state
+      - protocolVersion
+      - nonce
+      - options
+    properties:
+      ipAddress:
+        type: string
+        example: 127.0.0.1
+        description: IPv4 address of the peer node.
+      port:
+        type: integer
+        example: 8001
+        minimum: 1
+        maximum: 65535
+        description: The port the peer node uses for P2P communication.
+      protocolVersion:
+        type: string
+        example: 1.1
+        description: The protocol version of Lisk Core that the peer node runs on.
+      networkID:
+        type: string
+        example: 'k9AP5b5w2Q564keTai59g7UICcebc/oUKF8CyEI0iz4='
+        description: Base64 encoded value of the network identifier as per LIP-0009.
+      state:
+        type: string
+        example: 'connected'
+        enum:
+          - connected
+          - disconnected
+        description: |
+          The state of the peer.
+      options:
+        type: object
+        description: Optional field defined by application
+
+  NodeInfo:
+    type: object
+    required:
+      - version
+      - protocolVersion
+      - networkID
+      - lastBlockID
+      - height
+      - chainMaxHeightFinalized
+      - syncing
+      - unconfirmedTransactions
+      - genesisConfig
+    properties:
+      version:
+        type: string
+        format: version
+        example: v0.8.0
+        description: The application version that the node is running on.
+      protocolVersion:
+        type: string
+        example: 1.1
+        description: The protocol version of Lisk Core that the peer node runs on.
+      networkID:
+        type: string
+        example: 'k9AP5b5w2Q564keTai59g7UICcebc/oUKF8CyEI0iz4='
+        description: Base64 encoded value of the network identifier as per LIP-0009.
+      lastBlockID:
+        description: ID of the last processed block.
+        type: string
+        example: vPFBr0ZTsyP9hUwKaQD8dW2lKBpgMWkRdgrTUUmBBCE=
+      height:
+        type: integer
+        example: 123
+        minimum: 1
+        description: |
+          Current block height of the node.
+          Represents the current number of blocks in the chain on the node.
+      chainMaxHeightFinalized:
```

finalized height will always refer to the highest value, so it can be just used as finalizedHeight. 
```suggestion
      finalizedHeight:
```

---

I think this was the conclusion of what we called this value in the endpoint.

---

Conclusion where? I don't remember any discussion around this topic.

---

the name is exactly the same as in LIP https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#computing-prevotes-and-precommits.
and when this property was added, it was discussed at that time if I remember correctly